games.json

```diff
@@ -8334,24 +8348,31 @@
     ],
     "updates": [],
     "notes": [],
-    "storeIds": {},
-    "dateChanged": "2024-07-25T21:21:20.558Z"
+    "storeIds":  {
+      "steam": "2668080"
+    },
+    "dateChanged": "2024-07-31T00:00:00.000Z"
   },
   {
     "url": "https://snowbreak.amazingseasun.com/",
-    "slug": "snowbreak-containment-zone-chn",
-    "name": "Snowbreak: Containment Zone (CHN)",
+    "slug": "snowbreak-containment-zone-china",
+    "name": "Snowbreak: Containment Zone (CN)",
```

Comment:
Just nitpicking, but I don't think this was that big of a deal to change, but it's fine to stay.

Reply:
@Starz0r Yeah this is fine.